dist/icon-ellipsis-vertical-vimeo.css

```diff
@@ -0,0 +1,28 @@
+.icon-ellipsis-vertical-vimeo-navy {
```

Comment:
should all the svg references for the various colors point to the same svg?

Reply:
I notice that in `dist/icon-ellipsis-vimeo.css` that each class, which corresponds to a color, has it's own svg url.